src/Nethermind/Nethermind.AuRa/Contracts/SystemContract.cs

```diff
@@ -54,9 +54,23 @@ protected Transaction GenerateTransaction(byte[] transactionData, long gasLimit
         
         public void InvokeTransaction(BlockHeader header, ITransactionProcessor transactionProcessor, Transaction transaction, CallOutputTracer tracer)
         {
-            if (transaction != null && !TryInvokeTransaction(header, transactionProcessor, transaction, tracer))
+            if (transaction != null)
             {
-                throw new AuRaException($"System call returned error '{tracer.Error}' at block {header.Number}.");
+                bool failure;
+                
+                try
+                {
+                    failure = !TryInvokeTransaction(header, transactionProcessor, transaction, tracer);
+                }
+                catch (Exception e)
+                {
```

Comment:
usually ine does not expect the Try method to throw an exception

Reply:
True, rewriting...